itkwidgets/_initialization_params.py

```diff
@@ -22,7 +22,7 @@ def init_params_dict(itk_viewer):
         'gradient_opacity': itk_viewer.setImageGradientOpacity,
         'gradient_opacity_scale': itk_viewer.setImageGradientOpacityScale,
         'interpolation': itk_viewer.setImageInterpolationEnabled,
-        'gaussians': itk_viewer.setImagePiecewiseFunctionGaussians,
+        'piecewise_function_points': itk_viewer.setImagePiecewiseFunctionPoints,
```

Comment:
How about `tf_points` or something else shorter here?

Reply:
Or the more specific `transfer_function_points`?

Reply:
Changed to `transfer_function_points`

Reply:
@PaulHax thank you! How about `transfer_function` for brevity? I do not think `_points` is adding enough information to warrant the length for interactive use. They are always points and this is implicit in the specification.

Reply:
Better and better.  Changed.